### ISSUE TITLE
[dediverge] friendly logs

### ISSUE THREAD

**Issue by olets** [olets/git-replay#3]
**rebase**, **rebase-onto**, and **stage** all log friendly messages before and after taking an action, and after all actions are done

```shell
% git replay
Rebasing feature off main # in blue
git rebase --rerere-autoupdate main feature
Successfully rebased and updated refs/heads/feature
Replay complete # in blue
````

**dediverge** does not

```shell
% git replay dediverge --force
Checking the replay config for diverged branches...
The following branches have diverged from their remote tracking branches:
feature
feature2
git checkout feature
# …
git checkout feature2
#
```

That should read

```shell
% git replay dediverge --force
Checking the replay config for diverged branches...  # in blue <---- changed
The following branches have diverged from their remote tracking branches:
feature
feature2
Dediverging feature # in blue <---- changed
git checkout feature
# …
Successfully dediverged feature #  <---- changed
Dediverging feature2 # in blue <---- changed
Dediverging feature
git checkout feature2
#
Successfully dediverged feature2 #  <---- changed
Replay complete # in blue <---- changed
```

**Comment by olets** [comment]
Did the above except for

```
Successfully dediverged feature2
```

In the rebase example that message is from git-rebase, not git-replay.

**Comment by olets** [comment]
Closed by [v3.1](https://github.com/olets/git-replay/releases/tag/v3.1)
